cnxdb/archive-sql/schema/tree_to_json.sql

```diff
@@ -37,7 +47,7 @@ SELECT
            ELSE '},' END
       AS "toc"
 FROM t left join  modules m on t.value = m.module_ident
-    WINDOW w as (ORDER BY corder) order by corder ) tree ;
+    WINDOW w as (ORDER BY corder) order by corder ) tree
```

Comment:
when `;` is removed here should not it also be removed in this later loc as well? https://github.com/openstax/cnx-db/blob/34d11bfc3ce5be6bf35d4d8312729a6ccc89aec4/cnxdb/archive-sql/schema/tree_to_json.sql#L87

Reply:
From what I can tell it's not a huge problem since we are only including one statement in the function. But it would probably be good to add `;` back in.